### PR TITLE
docs: Update RHEL installation guide

### LINKED ISSUE #415: update RHEL documentation
Remove warning and update obs url

### PR DESCRIPTION
**Pull request by GabyCT**
Remove the warning as we currently we are able to test RHEL and update
the correct OBS url.

Fixes #415

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: install/rhel-installation-guide.md [hunk 1]
```diff
@@ -1,19 +1,11 @@
...
->
->   See https://github.com/kata-containers/ci/issues/3 for further details.
-
 1. Install the Kata Containers components with the following commands:
 
    ```bash
    $ source /etc/os-release
    $ ARCH=$(arch)
-   $ sudo -E yum-config-manager --add-repo "http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/RHEL_${VERSION_ID}/home:katacontainers:releases:${ARCH}:master.repo"
+   $ sudo -E yum-config-manager --add-repo "http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/RHEL_7/home:katacontainers:releases:${ARCH}:master.repo"
```
Comment (jodh-intel) [inline install/rhel-installation-guide.md]: Why is this change required? We really don't want to hard-code the `7` in there (RHEL 8 is in beta...).

File: install/rhel-installation-guide.md [hunk 2]
```diff
@@ -1,19 +1,11 @@
 # Install Kata Containers on RHEL
 
-> **Warning:**
->
-> - The RHEL packages are provided as a convenience to users until native
->   packages are available in RHEL. However, they are **NOT** currently tested
->   (although CentOS is) so caution should be exercised.
->
->   See https://github.com/kata-containers/ci/issues/3 for further details.
-
```
Comment (jodh-intel) [inline install/rhel-installation-guide.md]: I'd like to see this warning removed, but aren't we still blocked on the following?

- https://github.com/kata-containers/ci/issues/3
- https://github.com/kata-containers/ci/issues/129

File: install/rhel-installation-guide.md (hunk 1 above)
Comment (GabyCT) [reply install/rhel-installation-guide.md]: @jodh-intel that is the OBS url, ID is equal to "rhel" without including a number or version

File: install/rhel-installation-guide.md (hunk 2 above)
Comment (GabyCT) [reply install/rhel-installation-guide.md]: @jodh-intel , I already open a PR in tests for installation scripts, I am waiting there for a review

File: install/rhel-installation-guide.md (hunk 1 above)
Comment (GabyCT) [reply install/rhel-installation-guide.md]: @jodh-intel version id does not match the obs url

Comment (GabyCT) [reply install/rhel-installation-guide.md]: cat /etc/os-release
NAME="Red Hat Enterprise Linux Server"
VERSION="7.6 (Maipo)"
ID="rhel"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="7.6"
PRETTY_NAME="Red Hat Enterprise Linux Server 7.6 (Maipo)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:7.6:GA:server"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

File: install/rhel-installation-guide.md (hunk 2 above)
Comment (jodh-intel) [reply install/rhel-installation-guide.md]: OK. This needs to block on it then, so I'll ad a dnm label...

File: install/rhel-installation-guide.md (hunk 1 above)
Comment (jodh-intel) [reply install/rhel-installation-guide.md]: In which case, I'd do something like:

```sh
$ VERSION=$(echo $VERSION_ID|cut -d\. -f1)
```

Then, use `RHEL_${VERSION}` in the URL.
